web/src/routes.js

```diff
@@ -2,13 +2,15 @@ import NoMatch from './components/NoMatch';
 import CreateAccount from './containers/admin/CreateAccount';
 import ApdApplication from './containers/ApdApplication';
 import Login from './containers/Login';
+import Logout from './containers/Logout';
 
 import StateDash from './containers/StateDashboard';
 
 const routes = [
   { path: '/', component: StateDash, exact: true, nonPrivate: false },
   { path: '/apd', component: ApdApplication, exact: true, nonPrivate: false },
   { path: '/login', component: Login, nonPrivate: true },
+  { path: '/logout', component: Logout, nonPrivate: false },
```

Comment:
Make the `/logout` path private, so when you logout, you land on that page, get logged out, and immediately get routed back to `/login`